draft-group-privacypass-k-check.md

```diff
@@ -27,25 +27,52 @@ author:
     email: jdoe@weylandyutani.com
 
 normative:
+  PRIVACYPASS: I-D.ietf-privacypass-architecture
+  PRIVACYPASS-ISSUANCE: I-D.ietf-privacypass-protocol
+  CONSISTENCY: I-D.ietf-privacypass-key-consistency
+  OHTTP: I-D.ietf-ohai-ohttp
 
 informative:
+  DOUBLE-CHECK: I-D.schwartz-ohai-consistency-doublecheck
 
 
 --- abstract
 
-TODO Abstract
-
+This document describes a protocol called K-Check for implementing HTTP resource consistency checks.
+The primary use case for K-Check is for deployments of protocols such as Privacy Pass and Oblivious
+HTTP wherein privacy goals require that clients have a consistent view of some protocol-specific
+resource (typically, a public key).
 
 --- middle
 
 # Introduction
 
-This document describes a protocol mechanism based on {{?DoubleCheck=I-D.schwartz-ohai-consistency-doublecheck}}
-for checking that an HTTP resource is consistent with the view of one or more mirrors. In this
-context, a mirror is an HTTP server that fetches and caches copies of an HTTP resource
+Privacy-enhancing protocols such as Privacy Pass {{PRIVACYPASS}} and Oblivious HTTP {{OHTTP}}
+require clients to obtain and use a public key for execution. In Privacy Pass,
+public keys are used by clients when issuing and redeeming tokens for anonymous
+authorization. In Oblivious HTTP (OHTTP), clients use public keys to encrypt
+messages to a gateway server.
+
+Deployments of protocols such as Privacy Pass and OHTTP requires that many, if not all,
+clients share the same key. This is because the privacy properties depend on the client
```

Comment:
```suggestion
Deployments of protocols such as Privacy Pass and OHTTP requires that very large sets of clients
share the same key, or even that all clients globally share the same key. This is because the privacy properties depend on the client
```